### PR TITLE
fix(DB/creature_template): Ulduar - Overload

### LINKED ISSUE #3589: (Raid - Ulduar) Iron Council "Overload Visual"
##### CURRENT BEHAVIOUR:
In the 10 and 25-man version of Iron Council in Ulduar, the "Overload Visual" NPC can be targeted and attacks players while it is summoned.
WoWScrnShot_101820_212903

##### EXPECTED BLIZZLIKE BEHAVIOUR:
Overload Visual NPC should not take part in combat or be targetable.

##### STEPS TO REPRODUCE THE PROBLEM:
Enter 10 or 25 lockout of Ulduar
.go xyz 1646.1 119.7 427.3 603 3.1
Wait for Stormcaller Brundir to cast Overload
/target Overload Visual
Check combat log for "Overload Visual" damage

##### AC HASH/COMMIT:
ab5933e

##### OPERATING SYSTEM:
Win 10

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/93554103-raid-ulduar-iron-council-overload-visual?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Si1ker**
This change will fix the visual overload npc at the Iron assembly encounter

## CHANGES PROPOSED:
-  Change AI for the NPC "Overload Visual"
-  Update unit_flag

## ISSUES ADDRESSED:
- Closes #3589 

## TESTS PERFORMED:
- Tested in-game
image

## HOW TO TEST THE CHANGES:

.go xyz 1626.17 120.07 427.27 603 3.13
Engage combat and wait for the Overload cast from Brundir
The NPC visual overload won't be visible/active anymore

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by aradep** [comment]
Tested, works :)
image

File: data/sql/updates/pending_db_world/rev_1603072521176787100.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1603072521176787100');
+
+/*
+ * Update by Silker | <www.azerothcore.org> | Copyright (C)
+*/
+
+UPDATE `creature_template` SET `unit_flags`=33554432, `AIName`='PassiveAI' WHERE  `entry` IN
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1603072521176787100.sql:7]: no. Add flags, dont override them.

File: data/sql/updates/pending_db_world/rev_1603072521176787100.sql [hunk 2]
```diff
@@ -4,6 +4,6 @@ INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1603072521176787100');
  * Update by Silker | <www.azerothcore.org> | Copyright (C)
 */
 
-UPDATE `creature_template` SET `unit_flags`=33554432, `AIName`='PassiveAI' WHERE  `entry` IN
+UPDATE `creature_template` SET `unit_flags`=`unit_flags`|33554432, `AIName`='PassiveAI' WHERE  `entry` IN
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1603072521176787100.sql:7]: Still no.
You add every flag you want to add by them self like this

unit=unit|flag1|flag2|flag3

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1603072521176787100.sql:7]: just add the flag you're adding, not all flags

Comment (Si1ker) [reply data/sql/updates/pending_db_world/rev_1603072521176787100.sql:7]: That's a single flag  ._.

```sql
33554432 | 0x0200 0000 | UNIT_FLAG_NOT_SELECTABLE | Can't be selected by mouse or with /target {name} command.
```

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1603072521176787100.sql:7]: ok

**Comment by FrancescoBorzi** [comment]
@Si1ker the build is failing, there are startup errors. Please fix them.

File: data/sql/updates/pending_db_world/rev_1603072521176787100.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1603072521176787100');
+
+/*
+ * Update by Silker | <www.azerothcore.org> | Copyright (C)
+*/
+
+UPDATE `creature_template` SET `AIName`='PassiveAI' WHERE `entry`=32866;
+UPDATE `creature_template` SET `unit_flags`=`unit_flags`|33554432 WHERE  `entry` IN
+(32866, 33690;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1603072521176787100.sql:9]: Missing closing bracket

Comment (Si1ker) [reply data/sql/updates/pending_db_world/rev_1603072521176787100.sql:9]: @Kitzunu  ?

Comment (Si1ker) [reply data/sql/updates/pending_db_world/rev_1603072521176787100.sql:9]: Shin >:c  😹   oks

**Review by FrancescoBorzi — APPROVED** [review]

**Review by Helias — APPROVED** [review]

**Pull request merged**